src/Raven.Server/ServerWide/Commands/Subscriptions/PutSubscriptionBatchCommand.cs

```diff
@@ -30,4 +32,28 @@ public override DynamicJsonValue ToJson(JsonOperationContext context)
             return djv;
         }
     }
+
+    public class PutSubscriptionBatchCommand : PutSubscriptionBatchCommandBase<PutSubscriptionCommand>
+    {
+
+        public PutSubscriptionBatchCommand()
+        {
+        }
+
+        public PutSubscriptionBatchCommand(List<PutSubscriptionCommand> commands, string uniqueRequestId) : base(commands, uniqueRequestId)
+        {
+        }
+    }
+
+    public class PutShardedSubscriptionBatchCommand : PutSubscriptionBatchCommandBase<PutShardedSubscriptionCommand>
```

Comment:
this is not a place for that, what is the benefit of having this command?

Reply:
we need a way to recognize if we add a batch of sharded subscriptions or  a batch of regular in CSM